config/config.go

```diff
@@ -2,24 +2,42 @@ package config
 
 import (
 	"fmt"
-	"os"
 	"strings"
 	"syscall"
 
-	"github.com/BurntSushi/toml"
+	"github.com/ipfs/go-log"
 	"github.com/keep-network/keep-common/pkg/chain/ethereum"
+	ethereumCommon "github.com/keep-network/keep-common/pkg/chain/ethereum"
+	"github.com/keep-network/keep-core/pkg/beacon/registry"
+	ethereumChain "github.com/keep-network/keep-core/pkg/chain/ethereum"
+	ethereumBeaconGen "github.com/keep-network/keep-core/pkg/chain/ethereum/beacon/gen"
 	"github.com/keep-network/keep-core/pkg/net/libp2p"
+	"github.com/mitchellh/mapstructure"
+	"github.com/spf13/viper"
 	"golang.org/x/crypto/ssh/terminal"
 )
 
-// #nosec G101 (look for hardcoded credentials)
-// This line doesn't contain any credentials.
-// It's just the name of the environment variable.
-const passwordEnvVariable = "KEEP_ETHEREUM_PASSWORD"
+var logger = log.Logger("keep-config")
+
+// Environment variables names for properties configuration. For simplification we
+// bind the names that are different from the auto-generated by viper with the
+// `viper.AutomaticEnv()` function. For example instead of expecting
+// `KEEP_ETHEREUM_ACCOUNT_KEYFILEPASSWORD` we expect `KEEP_ETHEREUM_PASSWORD`.
+const (
+	// #nosec G101 (look for hardcoded credentials)
+	// This line doesn't contain any credentials.
+	// It's just the name of the environment variable.
+	ethereumPasswordEnvVariable         = "KEEP_ETHEREUM_PASSWORD"
+	ethereumKeyFileEnvVariable          = "KEEP_ETHEREUM_KEYFILE"
+	libp2pPeersEnvVariable              = "KEEP_PEERS"
+	libp2pPortEnvVariable               = "KEEP_PORT"
+	libp2pAnnouncedAddressesEnvVariable = "KEEP_ANNOUNCED_ADDRESSES"
```

Comment:
I wonder if we shouldn't name them `KEEP_NETWORK_PEERS`, `KEEP_NETWORK_PORT`, and `KEEP_NETWORK_ANNOUNCED_ADDRESSES` and rename `[LibP2P]` section in `config.toml.SAMPLE` to `[Network]`. 

There are two reasons:
- The fact we use libp2p for networking is an implementation detail.
- We have metrics and diagnostics that also need to have the port configured. They would be `KEEP_METRICS_PORT` and `KEEP_DIAGNOSTICS_PORT`.